jcore-muc7-reader/src/main/java/de/julielab/jcore/reader/muc7/MUC7Reader.java

```diff
@@ -1005,13 +1007,13 @@ public void getNext(CAS cas) throws IOException, CollectionException {
 			startPosition = 0;
 			// important: keep this sequence!
 			annotateHeader(docNode);
-//			annotateSlug(docNode, textToBeProcessed);
-//			annotateDate(docNode, textToBeProcessed);
-//			annotateNumOfWords(docNode, textToBeProcessed);
-//			annotatePreamble(docNode, textToBeProcessed);
+			annotateSlug(docNode, textToBeProcessed);
+			annotateDate(docNode, textToBeProcessed);
+			annotateNumOfWords(docNode, textToBeProcessed);
+			annotatePreamble(docNode, textToBeProcessed);
 			annotateText(docNode, textToBeProcessed);
-//			annotateTrailer(docNode, textToBeProcessed);
-			annotateCorefs();
+			annotateTrailer(docNode, textToBeProcessed);
+//			annotateCorefs();
```

Comment:
Sorry, I think this happened because I was using 2.1.x-bugfixes. The 
line not be uncommented.

On 06/20/2016 01:01 PM, Franz Matthies wrote:
>
> In 
> jcore-muc7-reader/src/main/java/de/julielab/jcore/reader/muc7/MUC7Reader.java 
> <https://github.com/JULIELab/jcore-base/pull/19#discussion_r67671472>:
>
> >  			annotateText(docNode, textToBeProcessed);
> > -//			annotateTrailer(docNode, textToBeProcessed);
> > -			annotateCorefs();
> > +			annotateTrailer(docNode, textToBeProcessed);
> > +//			annotateCorefs();
>
> Thanks for the pull request.
> Why is this line uncommented? Where are the coreferences added to the 
> CAS if not here?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/JULIELab/jcore-base/pull/19/files/dbebac744624e945bffd3f27ede97bd4b9fac972#r67671472>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/AGy_ProczIA6KbWUe77laJoU3tzCFhe8ks5qNnMKgaJpZM4I5hsy>.
>